### ISSUE TITLE
Multi agency and apiKeys relationship

### ISSUE THREAD

**Issue by Raven888888** [TheTransitClock/transitime#273]
In the Multi Agency wiki setup guide, I follow [Step 4](https://github.com/TheTransitClock/transitime/wiki/Multi-Agency-Configuration#4-create-api-key) to create
- 1 api key (`abc123`) for my `primary` agency
- 1 api key (`def456`) for my `halifax` secondary agency

WebAgency data for both are stored in the primary agency database only.

Then, I start tomcat for API and WEB, using 
```
JAVA_OPTS="-Dtransitclock.apikey=abc123 -Dtransitclock.configFiles=/home/ubuntu/transitime/primary.properties"
```
as I understand it only requires 1 instance of API and WEB.

*My question is, why do I need 2 api keys if only the primary key is passed to Tomcat?*

I can curl
```
api/v1/key/abc123/agency/primary/command/routes?format=json
api/v1/key/abc123/agency/halifax/command/routes?format=json
```
and get the correct results. 

_I expect halifax agency should require its own apikey, but it does not look like this is the case???_

And when I curl
```
api/v1/key/def456/agency/primary/command/routes?format=json
api/v1/key/def456/agency/halifax/command/routes?format=json
```
it shows key is not valid.

Does this mean all agencies essentially share the same apiKey?

PS: obviously the apiKey is obfuscated.

**Comment by vesavlad** [comment]
What's the end goal that you try to archive here? 
Running one transitclock but having specialized view for each agency that you have there?

**Comment by Raven888888** [comment]
Yes @vesavlad , and one apiKey should only see the corresponding agency data
